docs/sources/administration/api-keys/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 'API keys in Grafana'
+menuTitle: 'API keys'
+description: 'This contains information about API keys in Grafana'
+aliases: [docs/sources/administration/api-keys/_index.md]
+weight: 300
+keywords:
+  - Api keys
+  - Service accounts
+---
+
+# API keys in Grafana
+
+An API key is a encrypted string that identifies as a specific role to your Grafana instance when talking to APIs. You can more easily use Grafana HTTP APIs by leveraging API keys.
```

Comment:
Not entirely sure what the first sentence means. What does 'identifies as a specific role' mean? And let's find another word for 'talking'. Do you mean connecting?

Reply:
i have updated to "API Keys can be used to interact with Grafana HTTP APIs"